ocaml/xapi/xapi_crashdump.ml

```diff
@@ -11,22 +11,15 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
  *)
-exception Not_implemented
-
-let nothrow f () = try f() with _ -> ()
-
-let create ~__context ~vM ~vDI =
-  let cdumpref = Ref.make() in
-  let uuid = Uuid.to_string (Uuid.make_uuid()) in
-  Db.Crashdump.create ~__context ~ref:cdumpref ~uuid ~vM ~vDI ~other_config:[];
-  cdumpref
 
 let destroy ~__context ~self =
```

Comment:
Where do we use this? Can't we just move the function in an helper module and get rid of this module altogether?